src/main/java/io/javalin/core/JavalinServlet.kt

```diff
@@ -146,11 +130,7 @@ class JavalinServlet(
         LogUtil.logRequestAndResponse(ctx, logLevel, matcher, log, gzipShouldBeDone(ctx))
     }
 
-    private fun shouldCheckForStaticFiles(endpointEntries: List<HandlerEntry>, type: HandlerType, requestUri: String) = when {
-        type == HandlerType.GET && endpointEntries.isEmpty() -> true
-        type == HandlerType.HEAD && matcher.findEntries(HandlerType.GET, requestUri).isEmpty() -> true
-        else -> false
-    }
+    private fun getIsMapped(requestUri: String) = matcher.findEntries(HandlerType.GET, requestUri).isNotEmpty()
```

Comment:
isGetMapped?

Reply:
That's better. I normally follow strict camelCasing but maybe `isGETMapped` could work?

Reply:
I'd say that 'Get' is just verb, no abbrev no special meaning so it does not deserve to be highlighted like this :)
I have no preference though, do one that you like better.

Reply:
isGetMethodMapped would provide the meaning if Get is too ambiguous

Reply:
Both `get` and `method` are pretty unfortunate words to use in a method name, but I guess it's somewhat clear from the context. I'll use `isGetMethodMapped`.